src/ProfilesPage/ProfilePage.js

```diff
@@ -141,7 +159,7 @@ class ProfilePage extends Component {
   suspendUserButton() {
     const suspended = this.props.userInfo.suspended;
     const admin = sessionStorage.getItem("isAdmin");
-    if (this.state.suspendedState) {
+    /*if (this.state.suspendedState) {
```

Comment:
Do we not want unsuspend buttons anymore?

Reply:
done